packages/ui/src/data/navbar-menu-data.ts

```diff
@@ -1,57 +1,59 @@
 import { MenuGroupType, MenuGroupTypes } from '@components/navbar/types'
 
-export const navbarMenuData: MenuGroupType[] = [
+import { i18nextViewsInstance } from '../i18n/i18n'
```

Comment:
Can't we use a single '../i18n' import  instead of '../i18n/i18n'?
Need to change the folder structure with i18n/index.ts instead of i18n/i18n.ts